package.json

```diff
@@ -55,6 +55,7 @@
     "@canonical/jujulib": "2.0.0-beta.8",
     "@canonical/macaroon-bakery": "1.2.2",
     "@canonical/react-components": "0.37.6",
+    "@reduxjs/toolkit": "1.9.0",
```

Comment:
This was mostly just added to provide additional Redux types (I've also made use of the toolkit features in another branch).